documentation/adr/0002-use-github-pages-for-user-documentation.md

```diff
@@ -0,0 +1,35 @@
+[<-previous](0001-record-architecture-decisions.md) | [next->](0003-pace-projects-will-have-independent-documentation.md)
+
+# 2. Use GitHub Pages for user documentation
+
+Date: 2020-Jun-20
+
+## Status
+
+Accepted
+
+## Context
+
+A consistent platform is required for user documentation from the PACE projects: Brille, Euphonic and Horace/Herbert.
+
+Two platforms support, are well used for this service:
+
+- [Read the Docs](https://readthedocs.org/)
+- [GitHub pages](https://pages.github.com/)
+
+Both platforms will display documentation built by [Sphinx](https://www.sphinx-doc.org/) from reStructuredText source files.
+
+Brille includes compiled C libraries that contribute APIs to the project. The Read the Docs build/deploy pipeline does not support inclusion of documentation generated from C source.
+
+## Decision
+
+We will use GitHub pages for all PACE project documentation.
+
+## Consequences
+
+- documentation will be available at URLs under `pace-neutrons.github.io` 
+- GitHub pages can be configured to make the documentation available at any owned URL
+- documentation is stored "as code" alongside the source
```

Comment:
Does 'as code' mean the .rst or html?

Reply:
Yes.

Reply:
There are two cases here:

- The document source (i.e. `rst` and static `html` is part of the 'codebase`. 
- The built artefacts (`html`, `css`, ...) will be stored in the same GitHub repository, either in the `gh_pages` branch or a configured "built documentation" folder.

Reply:
Gotcha, it might be helpful to specify that 'code' means rst and static html. And I guess the rest will be documented in another ADR once we've decided exactly where to store the versioned build artefacts.

Reply:
I should also an an extra ADR documenting the preference to use the `gh_pages` branch for built documentation and the consequences for branch and build management that creates.